auth0/auth0-authorization-extension.js

```diff
@@ -0,0 +1,59 @@
+/*
+*  This rule been automatically generated by auth0-authz-extension
+*  Updated by kerin.cosford@digital.justice.gov.uk, 2017-01-14T22:47:58.651Z
+ */
+function (user, context, callback) {
+  // If connection is not passwordless skip this rule
+  if (context.connection !== 'email') {
```

Comment:
`context.connection` is equal to `'email'` when using passwordless authentication.